### PR TITLE
IR-385: Update incident reporting service data model and URLs

### PR DESCRIPTION
**Pull request by ushkarev**
Depends on [hmpps-incident-reporting-api#139](https://github.com/ministryofjustice/hmpps-incident-reporting-api/pull/139)

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsApiMockServer.kt [hunk 1]
```diff
@@ -76,14 +76,14 @@ class IncidentsApiMockServer : WireMockServer(WIREMOCK_PORT) {
...
-        incidentNumber = nomisIncidentId,
+        reportReference = nomisIncidentId,
         type = ReportWithDetails.Type.ATTEMPTED_ESCAPE_FROM_ESCORT,
         incidentDateAndTime = "2021-07-05T10:35:17",
         prisonId = "ASI",
         title = "There was an incident in the exercise yard",
         description = "Fred and Jimmy were fighting outside.",
         event = Event(
-          eventId = "123",
+          eventReference = nomisIncidentId,
```
Comment (ushkarev) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/incidents/IncidentsApiMockServer.kt:86]: IRS api makes the ”Event” reference the same as that of the “Report”

**Review by karenmillermoj — DISMISSED** [review]
lgtm

**Comment by ushkarev** [comment]
@karenmillermoj sorry, but i rebased this cos i had to change the JSON file after you approved. feel free to merge when you’re happy (cos i don’t have merge powers)

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**